### DISCUSSION TITLE
24 seeds do not work

### DISCUSSION THREAD

**Discussion by sadegh1010** [Chia-Network/chia-blockchain#8670] [Support]
hey
recently,I reinstalled windows on main machine (version 1.1.2)  
install version 1.2.7 on new farmer machine and tried to do 24 seeds
wallet and everything else are OK , but for more than 600 plots I have this Error "has a farmer public key that is not in the farmer's pk list"
 only thing I have from plotter machine is 24 seeds
I have tried version 1.1.2 still nothing !
any idea??
should i re-plot?

**Comment by loppefaaret** [comment]
Sorry for getting to you this late, but did you manage to solve your issue ? it sounds like you perhaps somehow might have got your 24 words mixed up or something ? but before you replot all the hard work of creating 600 plots, lets try and figure out what happened ?

**Comment by sadegh1010** [comment]
On 7 Oct 2021 Thu at 15:05 loppefaaret ***@***.***> wrote:

> Sorry for getting to you this late, but did you manage to solve your issue
> ? it sounds like you perhaps somehow might have got your 24 words mixed up
> or something ? but before you replot all the hard work of creating 600
> plots, lets try and figure out what happened ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Chia-Network/chia-blockchain/discussions/8670#discussioncomment-1441830>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMXAFZIGZQGFLNWRMRFMWULUFWAWNANCNFSM5FE4EB6A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
hey
Well,I really thankful for your reply
I have 24 words. When I use them and sync wallet ,everything is fine except
plots.
I have errors on all of my plots
“Public farmer key , pk list “  same common error every one encountered
Unfortunately , I reinstalled windows and I don’t have backup of pervious
chia network app
I don’t what should I do


>

**Reply by loppefaaret** [reply]
ok, so it sounds like your plots have been created using a key, that you do not have the 24 words for anymore ? if that is the case, they are basically useless, im sorry to say

**Comment by sadegh1010** [comment]
On 8 Oct 2021 Fri at 18:54 loppefaaret ***@***.***> wrote:

> ok, so it sounds like your plots have been created using a key, that you
> do not have the 24 words for anymore ? if that is the case, they are
> basically useless, im sorry to say
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Chia-Network/chia-blockchain/discussions/8670#discussioncomment-1448936>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMXAFZI6OCEPEOWYLRJUKF3UF4EJDANCNFSM5FE4EB6A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
I have 24 words.
I don’t have mainnet backup
And plots don’t recognize with 24 words after syncing!

**Reply by loppefaaret** [reply]
if you try and list one of the plot files that are not being accepted as having `farmer public key that is not in the farmer's pk list`
list it with `chia plots check -n 5 -g X:\path\to\plots\plot-k32-2021-09-09-11-10-33-87n3vwvvsdbs7846yw448ac89aw.plot` or whatever the plot filename is, and take note of the "Farmer public key:" it reports
then do a `chia keys show` and see if it matches the "Farmer public key" reported under your key - if it does, that is the corret key - if it does not, it belongs to another key!

**Comment by BrandtH22** [comment]
@sadegh1010 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
